packages/server/tests/acceptance/htsPrecompile.spec.ts

```diff
@@ -239,6 +239,88 @@ describe('HTS Precompile Acceptance Tests', async function () {
     expect(tokenInfo.token.symbol).to.equal(TOKEN_SYMBOL);
   });
 
+  it('should be able to freeze and unfreeze fungible token transfers', async function() {
+    const baseHTSContractOwner = new ethers.Contract(BaseHTSContractAddress, BaseHTSJson.abi, accounts[0].wallet);
+
+    // expect the token to not be frozen
+    const txBefore = await baseHTSContractOwner.isFrozenPublic(HTSTokenContractAddress, accounts[0].wallet.address, { gasLimit: 1_000_000 });
```

Comment:
These 4 lines are pretty much repeated 2 times after this.
Let's just have a shared method that checks the frozen status